internal/core/application/feeder/service.go

```diff
@@ -0,0 +1,176 @@
+package feeder
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+
+	"github.com/tdex-network/tdex-daemon/internal/core/ports"
+)
+
+type Service struct {
+	repoManager ports.RepoManager
+	feederSvc   ports.PriceFeeder
+}
+
+func NewService(
+	repoManager ports.RepoManager,
+) Service {
+	return Service{repoManager: repoManager}
+}
+
+func (s *Service) Start(ctx context.Context) error {
```

Comment:
we don't have Start/Stop methods in the other app services at the moment.. they start doing their job usually by a `listenToWhatever` function like done [here](https://github.com/tdex-network/tdex-daemon/blob/v1/internal/core/application/wallet/service.go#L43-L44) or by registering handlers for events like [this one](https://github.com/tdex-network/tdex-daemon/blob/v1/internal/core/application/operator/service.go#L62-L63).

Let's keep this approach for now, then we will change all of them in a dedicated PR.
In this case, I don't think we need events so it's enough to add a `listenToPriceFeeds` method or something like that.

Also, you can see here how we need the missing `ListPriceFeeds` API and related ones in other layers (storage) so we can list them all and start only those that are not stopped (instead of all of them) without interacting with any repository.

Reply:
The feeder should also be stopped at interface layer as done for [other services](https://github.com/tdex-network/tdex-daemon/blob/v1/internal/interfaces/grpc/service.go#L320-L327). I can just agree with you if you say that this sìis wrong, but again let's make this kind of change in a dedicated PR to not create any mess.